### PR TITLE
add new task solution

### PR DESCRIPTION
**Pull request by Vladyslav-amp**

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,29 @@
...
+    <form>
+      <label class="field">
+        <input
+         type="search"
+         name="searchBar"
+         id="iconBig"
+         class="bigField"
+         placeholder="Try &quot;Los Angeles&quot;"
+         data-qa="big"
+         >
```
Comment (agniya-i) [inline src/index.html:23]:
```suggestion
        <input
         type="search"
         name="searchBar"
         id="iconBig"
         class="bigField"
         placeholder="Try &quot;Los Angeles&quot;"
         data-qa="big"
       >
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  line-height: 22px;
+  color: #3d4e61;
+  font-family: Avenir-bold, serif;
+  font-style: normal;
+}
+
+.smallField {
+  width: 100%;
+  height: 42px;
+  font-family: Avenir-normal, serif;
```
Comment (agniya-i) [inline src/style.css:45]: There are a some similar styles for `bigField` and `smallField`, let's improve this to not repeat these styles, and use the approach from the [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)

Screenshot 2023-07-10 at 20 57 02
Screenshot 2023-07-10 at 20 58 02

**Review by agniya-i — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form>
+      <label class="field fieldBig">
+        <input
+         type="search"
+         name="searchBar"
+         id="iconBig"
```
Comment (Viktor-Kost) [inline src/index.html:19]: try not use id's. Apply styles using class selectors
```suggestion
```

File: readme.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 # Search bar for Airbnb
 Replace `<your_account>` with your Github username and copy the links to Pull Request description:
-- [DEMO LINK](https://<your_account>.github.io/layout_search-bar-airbnb/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_search-bar-airbnb/report/html_report/)
+- [DEMO LINK](https://Vladyslav-amp.github.io/layout_search-bar-airbnb/)
+- [TEST REPORT LINK](https://Vladyslav-amp.github.io/layout_search-bar-airbnb/report/html_report/)
```
Comment (Viktor-Kost) [inline readme.md:4]: please add these links to PR description

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-bold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+form {
```
Comment (Viktor-Kost) [inline src/style.css:19]: use class selectors instead of tag ones

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+  background-size: 19px;
+  background-position: left 25px top 24px;
+}
+
+#iconSmall {
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px;
+  background-position: left 12px top 14px;
+}
```
Comment (Viktor-Kost) [inline src/style.css:82]: repeated styles. Use common selector (.icon) to move them there

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+.smallField {
+  height: 42px;
+  font-size: 14px;
+}
+
+.smallField:hover {
+  box-shadow: 0 3px 8px 0 rgba(61, 78, 97, 0.2);
+}
+
+.smallField:focus {
```
Comment (Viktor-Kost) [inline src/style.css:60]: move repeated styles to common selector (.area)
And I'd recommend to change naming
Use
.input
.input--small
.input--big

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
As an additional to my comments please check previous mentor's remarks
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3375#pullrequestreview-1524585231
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3375#discussion_r1258653804
https://github.com/mate-academy/layout_search-bar-airbnb/pull/3375#discussion_r1258658628

If you need any help feel free to ask at fe_chat

File: readme.md (hunk 4 above)
Comment (volodymyr-soltys97) [inline readme.md:4]: Add these links to the PR description
image

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Need to fix it
image
image
image

**Pull request closed without merging**